aws/data_source_aws_lb_test.go

```diff
@@ -192,6 +225,77 @@ data "aws_lb" "alb_test_with_name" {
 `, lbName)
 }
 
+func testAccDataSourceAWSLBConfigOutpost(lbName string) string {
+	return fmt.Sprintf(`
+data "aws_outposts_outposts" "test" {}
+
+data "aws_outposts_outpost" "test" {
+  id = tolist(data.aws_outposts_outposts.test.ids)[0]
+}
+
+resource "aws_lb" "alb_test" {
+  name            = "%s"
+  internal        = true
+  security_groups = [aws_security_group.alb_test.id]
+  subnets         = [aws_subnet.alb_test.id]
+
+  idle_timeout               = 30
+  enable_deletion_protection = false
+
+  tags = {
+    TestName = "TestAccAWSALB_outpost"
+  }
+}
+
+resource "aws_vpc" "alb_test" {
+  cidr_block = "10.0.0.0/16"
+
+  tags = {
+    Name = "terraform-testacc-lb-data-source-outpost"
+  }
+}
+
+resource "aws_subnet" "alb_test" {
+  vpc_id                  = aws_vpc.alb_test.id
+  cidr_block              = "10.0.0.0/24"
+  availability_zone       = data.aws_outposts_outpost.test.availability_zone
+  outpost_arn             = data.aws_outposts_outpost.test.arn
```

Comment:
This might help fix the `terrafmt` CI failure:

```suggestion
  vpc_id            = aws_vpc.alb_test.id
  cidr_block        = "10.0.0.0/24"
  availability_zone = data.aws_outposts_outpost.test.availability_zone
  outpost_arn       = data.aws_outposts_outpost.test.arn
```